cl/opinion_page/views.py

```diff
@@ -523,6 +525,24 @@ def view_opinion(request: HttpRequest, pk: int, _: str) -> HttpResponse:
     )
 
 
+@ratelimit_if_not_whitelisted
+def view_summaries(request: HttpRequest, pk: int, slug: str) -> HttpResponse:
+    cluster = get_object_or_404(OpinionCluster, pk=pk)
+
+    return render(
+        request,
+        "view_opinion_summaries.html",
+        {
+            "title": "%s, %s"
+            % (trunc(best_case_name(cluster), 100), cluster.citation_string),
```

Comment:
Looks like you got this from `view_opinion`. Would you mind abstracting it out from there and here and making a new function next to the `make_docket_title` function?